spec/factories/address.rb

```diff
@@ -1,6 +1,19 @@
 # frozen_string_literal: true
 
 FactoryBot.define do
+  address_types = WasteExemptionsEngine::TransientAddress.address_types
```

Comment:
Should this be `WasteExemptionsEngine::Address.address_types`? I know they're unlikely to be different but feels like we shouldn't be using TransientAddress in here.

Reply:
I think this is simply a copy paste error. Fixed in 65644a9.